agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1989,6 +1990,50 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
     // This is something that cannot be done at this moment because the MediaType field
     // must be computed with information obtained through NL80211_CMD_GET_WIPHY command of DWPAL,
     // which is currently not supported. See "Send standard NL80211 commands using DWPAL #782"
+    // For now, fill from radio data with dummy values based on information we have.
```

Comment:
It looks like #900 is about to be merged, so I guess it makes more sense to rebase on that instead of adding dummy code. The TODO should mention #900 and not the closed issue in case you insist on not rebasing.

Reply:
From what I can see, #900 does not contain relevant code for getting the info we need here.

Reply:
#900 implements NL80211_CMD_GET_STATION only and NL80211_CMD_GET_WIPHY is required here however. Now that the design for the nl80211 standard commands support has been validated, adding the NL80211_CMD_GET_WIPHY command is the next task I'll be working on.

Reply:
#900 provides the infrastructure for sending NL standard commands - if this does not provide you what you need to get the information you need, then what is? Is there an open issue for that?

Reply:
Got it - thanks @mariomaz , is there an open issue for the NL80211_CMD_GET_WIPHY support? If not, can you create one?
@morantr please add a comment and a task to replace the dummy values once @mariomaz adds the support.

Reply:
Created "[TASK] Add NL80211_CMD_GET_WIPHY command #967"

Reply:
> Created "[TASK] Add NL80211_CMD_GET_WIPHY command #967"

Add "Enabler" and "Certification" labels